wlauto/common/android/device.py

```diff
@@ -508,17 +509,18 @@ def get_properties(self, context):
     def _get_android_properties(self, context):
         props = {}
         props['android_id'] = self.get_android_id()
-        buildprop_file = os.path.join(context.host_working_directory, 'build.prop')
-        if not os.path.isfile(buildprop_file):
-            self.pull_file('/system/build.prop', context.host_working_directory)
-        self._update_build_properties(buildprop_file, props)
-        context.add_run_artifact('build_properties', buildprop_file, 'export')
-
+        self._update_build_properties(props)
+        
         dumpsys_target_file = self.path.join(self.working_directory, 'window.dumpsys')
         dumpsys_host_file = os.path.join(context.host_working_directory, 'window.dumpsys')
         self.execute('{} > {}'.format('dumpsys window', dumpsys_target_file))
         self.pull_file(dumpsys_target_file, dumpsys_host_file)
         context.add_run_artifact('dumpsys_window', dumpsys_host_file, 'meta')
+        
+        prop_file = os.path.join(context.host_working_directory, 'props.json')
```

Comment:
Maybe call it `'android-props.json'` -- would make it more clear for people browsing WA results who aren't necessarily familiar with WA.